package.json

```diff
@@ -57,7 +57,7 @@
     "data-spreadsheet"
   ],
   "dependencies": {
-    "moment": "2.18.1",
+    "moment": "^2.20.1",
```

Comment:
Previously we had version locked by ourselves (without `^`). Did the bumping dependencies strategy change?

Reply:
Nope, my bad.